rmgpy/data/thermo.py

```diff
@@ -1470,8 +1520,8 @@ def estimateRadicalThermoViaHBI(self, molecule, stableThermoEstimator ):
         if not isinstance(thermoData_sat, ThermoData):
             thermoData_sat = thermoData_sat.toThermoData()
         
-        
-        if not stableThermoEstimator == self.computeGroupAdditivityThermo:
+        if not (stableThermoEstimator == self.computeGroupAdditivityThermo
+                or isinstance(stableThermoEstimator.__self__, MLEstimator)):
```

Comment:
For some reason that doesn't work, i.e., when I do that, then it enters that if-statement even though `stableThermoEstimator` is the thermo estimation method from `MLEstimator`. I think the reason is that you are trying to compare the bound method of the class `MLEstimator` to a method of an instance of that class and that doesn't work. You don't have that issue for `computeGroupAdditivityThermo` because `self` refers to the class instance.